src/components/NcInputField/NcInputField.vue

```diff
@@ -162,6 +163,8 @@ const input = useTemplateRef('input-key')
 
 const hasTrailingIcon = computed(() => props.showTrailingButton || props.success)
 
+const internalPlaceholder = computed(() => props.placeholder || (isLegacy ? props.label : ''))
```

Comment:
I feel it should be like this, to not have an empty attribute
```suggestion
const internalPlaceholder = computed(() => props.placeholder || (isLegacy ? props.label : undefined))
```